workflow/github.md

```diff
@@ -1,2 +1,138 @@
 # Working on GitHub
 
+This document outlines the rules to contribute and to create issues to a project on the organisation [Canonical-websites](https://github.com/canonical-websites/)
+
+## Table of content
+
+* [How can I contribute?](#how-can-i-contribute-)
+  + [Report Bugs](#report-bugs)
+  + [Suggest Enhancements](#suggest-enhancements)
+  + [Work on an issue](#work-on-an-issue)
+* [Pull Requests](#pull-requests)
+  + [First contribution](#first-contribution)
+  + [Styleguides](#styleguides)
+    - [Commits messages](#commits-messages)
+    - [HTML](#html)
+    - [JS](#js)
+    - [Stylesheets](#stylesheets)
+    - [Python](#python)
+    - [Documentation](#documentation)
+  + [Merging process](#merging-process)
+* [Templates](#templates)
+  + [Issues template](#issues-template)
+  + [Pull-request template](#pull-request-template)
+
+## How can I contribute?
+
+In the Canonical web-sites organisation we work with Github. All types of work are tracked as GitHub issues. You will find in this section how you can contribute to this organisation.
+
+### Report Bugs
+
+Well done you have found a bug! Go to the corresponding project, it should be in the [Canonical websites organisation](https://github.com/canonical-websites/) and open an issue. These are the steps you need follow to add an issue:
+* **Title:** should describe your feature/bug
+* **Description:** fill this [template](#issues-template)
+* **Add labels:** see [Labels description](./labels.md)
+* **Submit your issue:** we will try to fix it as soon as possible!
+
+### Suggest Enhancements
+
+If you think details could change on our websites you can suggest enhancements by creating a new issue. These are the steps you need follow to suggest enhancements:
+* **Title:** should describe your feature/bug
+* **Description:** fill this [template](#issues-template)
+* **Screenshots:** Don’t forget to add screenshots of suggestion designs to explain
+* **Add labels:** see [labels description](./labels.md)
+* **Submit your issue:** we will try to fix it as soon as possible!
+
+### Work on an issue
+You are free to work on an issue if no one seems to work on it. Don’t forget to assign yourself to the issue. Once you finish the work open a [pull-request](#pull-Requests)
+
+## Pull Requests
+
+In this section you will find a description on how to create a pull-request for a Canonical Web-sites project. These are the steps to create a pull-request:
+* Fill the [template](#pull-request-template)
+* Make sure the tests are OK
+* Make sure the [styleguides](#styleguides) are respected
+* Add the [Labels](./labels.md)
+
+### First contribution
+
+If you want to create a pull-request you will need to fork the project you are working on. From your fork, create a branch and work on your issue.
+
+### Styleguides
+
+#### Commits messages
+
+The commit message must describe the feature you want to add to the project.
```

Comment:
I think this is so brief as to maybe add confusion. A commit will rarely describe a whole feature - as commits should be broken up into as small atomic pieces of work as possible. So this sentence could be read as suggesting that people submit exactly one commit per PR, which I don't think is good advice.

I don't think we're particularly precious about commits internally at the moment. This may change, but I think we should be organised internally before we hold external contributors to any particular standard.

I think we should work on a separate document about commits in a separate PR, and then when it's ready we could link to it from this section. But it shouldn't delay this PR in the mean-time.

So for now, could we just not mention commit messages in this PR until we actually have a policy of some sort?

Reply:
I understand. But i think this is something that should be added in the future. Commits messages are important and should have, even a small one, a practice document or section